test/sqllogictest/managed_cluster.slt

```diff
@@ -447,5 +447,23 @@ ALTER CLUSTER foo SET (MANAGED, SIZE '1', DISK)
 statement ok
 DROP CLUSTER foo
 
+
+simple conn=mz_system,user=mz_system
+ALTER SYSTEM SET enable_graceful_cluster_reconfiguration = true;
+----
+COMPLETE 0
+
+statement ok
+CREATE CLUSTER foo (SIZE '1')
+
+statement ok
+ALTER CLUSTER foo set (SIZE '2') WITH (WAIT FOR '0s')
+
+statement ok
+ALTER CLUSTER foo set (SIZE '4') WITH (WAIT UNTIL READY (TIMEOUT '0s', ON TIMEOUT 'COMMIT') )
+
+statement ok
+ALTER CLUSTER foo set (SIZE '4') WITH (WAIT UNTIL READY (TIMEOUT '10ms', ON TIMEOUT 'ROLLBACK') )
```

Comment:
Should we be asserting some particular state for the the cluster and replica system tables?

Reply:
idk...
Following the other tests in this file statement ok seems right. Most of the heavy lifting for testing this feature is in the cloudtest/test_managed_cluster.py.